polaris-react/.storybook/main.js

```diff
@@ -2,24 +2,13 @@ const path = require('path');
 const spawn = require('child_process').spawn;
 const CreateFileWebpack = require('create-file-webpack');
 const postcssPlugins = require('../config/postcss-plugins');
-// Enabling docs means the preview panel takes an extra 2ish seconds to load
-// This usually isn't a big deal, except when we're running all of our stories
-// through our a11y tests, and a 2s delay over several hundred stories adds up.
-// This is an escape hatch to disable docs only wheile we're running a11y tests
-const enableDocs = !parseInt(process.env.STORYBOOK_DISABLE_DOCS || '0', 10);
 
 module.exports = {
   core: {
     builder: 'webpack5',
   },
-  // Added to work around https://github.com/storybookjs/storybook/issues/15336
-  // The line below can be removed in @storybook/react v6.5.0
-  typescript: {reactDocgen: false},
   stories: ['../playground/stories.tsx', '../src/components/**/*/README.md'],
-  addons: [
-    {name: '@storybook/addon-essentials', options: {docs: enableDocs}},
```

Comment:
We'll want to keep `@storybook/addon-essentials`

Reply:
From what I can see this removes tabs and areas that aren't used. `Controls`, `Actions` and `Docs` 


| Before | After |
| --- | --- |
| ![Screen Shot 2022-07-27 at 6 32 45 AM](https://user-images.githubusercontent.com/19199063/181259928-b00b0661-2199-45c7-83e6-8b3acd6f5044.png) | ![Screen Shot 2022-07-27 at 6 32 52 AM](https://user-images.githubusercontent.com/19199063/181259962-36b11f51-e285-4a17-abb7-f3d4f1aeee08.png) |

Reply:
Oh crap `React.strictMode` is gone. I remember now. Adding this back.

Reply:
I think controls and viewport and measure would be useful too, and at that point I'd be happier including the kitchen sink of addon-essentials rather than opting in individually.